source/_components/climate.mysensors.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: "MySensors HVAC"
+description: "Instructions how to integrate MySensors climate into Home Assistant."
+date: 2016-09-14 18:20 +0100
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mysensors.png
+ha_category: Climate
+featured: false
+ha_release: 0.28+
+---
+
+Integrates MySensors HVAC into Home Assistant. See the [main component] for configuration instructions.
+
+The following actuator types are supported:
+
+##### MySensors version 1.5 and higher
+
+S_TYPE      | V_TYPE
+------------|-------------
+S_HVAC      | [V_HVAC_SETPOINT_HEAT, V_HVAC_SETPOINT_COOL, V_HVAC_FLOW_STATE, V_HVAC_SPEED]
```

Comment:
Maybe we should note that `V_HVAC_FLOW_STATE` is required to add the device in home assistant?